packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -275,6 +275,11 @@ export default {
         margin-top: 5px;
     }
 
+    .expandable-card > .card-collapse > hr {
+        margin-top: 0.25rem;
+        margin-bottom: 0.75rem;
```

Comment:
```suggestion
```
Any reason why `margin-bottom` is adjusted? Doesn't seem to play any role in achieving the outcome